docs/UserGuide.adoc

```diff
@@ -451,7 +460,7 @@ image:Help-exit/userguideurl.PNG[]
 
 TIP: Alternative methods: press kbd:[F1] or click on *Help* in the menu bar, and then click on *Help* in its sub menu.
 
-If you still remember about `add_expense` command but had forgotten about how to use it. You can view a simplified version of this
+If you still remember about `add_expense` command but had forgotten about how to use it. You can view a simplified help version of this
```

Comment:
If you just want to view the usage syntax of the `add_expense` command because you had forgotten the exact command syntax, you can enter *help add_expense* instead of *help*.